package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "digital-analytics-program",
+  "version": "4.0.0",
+  "description": "Provides a JavaScript file for US federal agencies to link or embed in their websites to participate in the Digital Analytics Program.",
+  "main": "Universal-Federated-Analytics.js",
+  "scripts": {
+    "build": "npx uglifyjs Universal-Federated-Analytics.js --output Universal-Federated-Analytics-Min.js",
+    "clean": "rm ./Universal-Federated-Analytics-Min.js",
+    "lint": "npx eslint Universal-Federated-Analytics.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/digital-analytics-program/gov-wide-code.git"
+  },
+  "license": "ISC",
```

Comment:
I didn't know the license was ISC. Where is this described?

Reply:
This was the default with npm init. I will change to public domain.

Reply:
@eddietejeda It may not be public domain either. @tdlowden What is the license for the code?